docs/introduction.md

```diff
@@ -5,402 +5,103 @@ This document is aimed at introducing developers to the Flow Data Provisioning S
    1. [Installation](#installation)
       1. [Dependencies](#dependencies)
-      2. [Manual Build](#manual-build)
+      2. [Build](#build)
    2. [Setting up a test environment](#setting-up-a-test-environment)
-6. [More Resources](#more-resources)
-   1. [Internal](#internal)
-   2. [External](#external)
+3. [More Resources](#more-resources)
 
 ## Getting Started
 
-The Flow Data Provisioning Service (DPS) is a web service that maintains and provides access to the history of the Flow [execution state](#execution-nodes).
+The Flow Data Provisioning Service (DPS) is a web service that maintains and provides access to the history of the Flow execution state.
 
 The reason for this need is that the in-memory execution state is pruned after 300 chunks, which makes it impossible to access the state history.
-Also, script execution is currently proxied from the [access nodes](#access-nodes) to [execution nodes](#execution-nodes), which is not scalable.
+Also, script execution is currently proxied from the access nodes to execution nodes, which is not scalable.
 The DPS makes access to the execution state _available_ (at any block height) and _scalable_ (so it does not infer load on the network nodes).
 
-Flow is often [upgraded with breaking changes](#sporks) that require a network restart. The new network with the updated version is started from a snapshot of the previous execution state.
-The final version of the previous execution state remains available through a legacy access node that connects to a legacy [execution node](#execution-nodes), but once again this is limited to the last 300 chunks.
+Flow is often upgraded with breaking changes that require a network restart. The new network with the updated version is started from a snapshot of the previous execution state.
+The final version of the previous execution state remains available through a legacy access node that connects to a legacy execution node, but once again this is limited to the last 300 chunks.
 
-## Flow
+### Indexing Past Sporks
 
-Most current blockchains are built as a homogenous system comprised of *full nodes*.
-Every full node collects and propagates the transaction it sees on the network.
-When being able to mine the next block, or being selected as the leader to propose the next block, they will choose the transactions to include according to the consensus rules.
-This means that every node needs to be able to execute every transaction it includes in a block in order to validate them.
-When validating other nodes' blocks, every node also needs to run every transaction to validate the block's correctness.
+In order to index past sporks using Flow DPS, three elements are needed from the spork to be indexed:
 
-These existing blockchain architectures thus suffer from well documented throughput limitations. Two approaches to mitigate this are layer-2 networks and sharding.
+* Its root checkpoint
+* The protocol state database from one of its consensus nodes
```

Comment:
No, we need tho protocol state database from an execution node. Otherwise, transactions and such are missing.